legacy/flash.h

```diff
@@ -24,9 +24,18 @@
 #include <stdlib.h>
 #include "secbool.h"
 
-#define FLASH_BYTE_ACCESS 1
+#define FLASH_BIT_ACCESS 1
+#define FLASH_BLOCK_WORDS 1
+
 #define FLASH_SECTOR_COUNT 24
 
+#define FLASH_BLOCK_SIZE (sizeof(uint32_t) * FLASH_BLOCK_WORDS)
+
+typedef uint32_t flash_block_t[FLASH_BLOCK_WORDS];
+
+#define FLASH_ALIGN(X) (((X) + 3) & ~3)
+#define FLASH_IS_ALIGNED(X) (((X)&3) == 0)
+
```

Comment:
Remove once the definitions are moved to flash_common.h.

Reply:
reverted in https://github.com/trezor/trezor-firmware/commit/87ca3a5654dd208defdaf5a63216d8ea748194e1